src/app/components/planning/planning.component.html

```diff
@@ -0,0 +1,2 @@
+<app-top-bar></app-top-bar>
```

Comment:
lets move app-top-bar to main component, so we wont have to duplicate it everywhere